scripts/gulpfiles/build_tasks.js

```diff
@@ -39,14 +40,14 @@ const {posixPath, quote} = require('../helpers');
 const PYTHON = process.platform === 'win32' ? 'python' : 'python3';
 
 /**
- * Suffix to add to compiled output files.
+ * Posix version of TSC_OUTPUT_DIR
  */
-const COMPILED_SUFFIX = '_compressed';
+const TSC_OUTPUT_DIR_POSIX = posixPath(TSC_OUTPUT_DIR);
 
 /**
- * Dependencies file (used by buildCompiled for chunking.
+ * Suffix to add to compiled output files.
  */
-const DEPS_FILE = path.join(BUILD_DIR, 'deps.js');
+const COMPILED_SUFFIX = '_compressed';
 
 /**
  * Name of an object to be used as a shared "global" namespace by
```

Comment:
Is this comment still accurate after your changes to chunking?

Reply:
I believe so.

This PR changes only the way the command line arguments to Closure Compiler are computed, not the actual arguments themselves—with two exceptions:

1. As noted, the chunks now contain some additional `.js` files that are empty except for `export {}`, caused by `.ts` files containing only `interface` declarations.  `closure-calculate-chunks` would previously omit these because they are not imported from anywhere, and it knew this based on the `deps.js` file generated by `closure-make-deps`.  This adds a tiny bit of bloat to the `_compressed.js` files, which could be fixed by having the new `getChunks` function check be smarter:
   -  Minimal improved smartness: just omit files any file containing `export {}`.
   - Perfect smartness: recursively parse the entrypoint file and all the files it imports, and only include files seen (and not seen in a previous chunk).  But then we are essentially reimplementing `closure-make-deps` + `closure-calculate-chunks` (though we could probably make it much faster for our specific use case).
0. Within each chunk, the order in which the files are listed on the Closure Compiler command line has changed.  This probably has knock-on effects to the order in which code appears in the `_compressed.js` file.

As part of testing this I compared the Closure command line arguments before and after this PR; modulo trivial differences (file order within chunks, and paths starting from `/Users/cpcallen/src/blockly/` vs. `./`) the arguments were the same, and when I sorted the files the generated `_compressed.js` files were byte-for-byte identical except for the added `var module$core$interfaces$i_foo={};` declarations for each empty interface modules discussed above.